### ISSUE TITLE
bug: Execution sustained for a few seconds after promise resolution.

### ISSUE THREAD

**Issue by ChlodAlejandro** [PassTheMayo/minecraft-server-util#27]
**Describe the bug:**
Given the JS file in the "Code" section, the library will temporarily hold the script open even after resolving the promise which returns the server status. For one-shot scripts like these, the delay can cause a bit of unneeded latency.

**Code:**
```javascript
const util = require('minecraft-server-util');
 
(async () => {
    try {
        const response = await util.status("localhost");
        console.log(JSON.stringify(response));
    } catch (error) {
        console.log(JSON.stringify({
            message: error.message || "Unknown error during execution.",
            ...error
        }));
    }
})();
```

**Expected behavior:**
Immediate termination of script execution after promise is resolved (program should exit right after `console.log` calls). Instead, the script is left open a short while (around 3 to 6 seconds) after either `console.log`.

**Additional context**
Something might be messing with the Node.js event loop that causes the script to be held open for longer than expected. The temporary workaround here is to place a `process.exit()` at the end of the try-catch block, however I'm not informed enough of how the library works in order to determine whether or not this causes issues with the connection between the Minecraft server and the script (since it would technically be a hard exit instead of a "graceful" one). For now, it works as a way to quickly wrap things up when needed.

**Comment by PassTheMayo** [comment]
I've done some digging and it appears to be the timeout hackery that I have made not properly clearing the timeout when the response has been returned. I'll get it fixed sometime this week and release a fix.

**Comment by ChlodAlejandro** [comment]
Got it. Thank you very much!

**Comment by PassTheMayo** [comment]
I had some free time so I got to work on fixing this. Just released version `3.2.0`, try updating and report back if the issue still persists. I had this same issue on my end but assumed it was Mocha hanging until you brought this to my attention.
